contracts/Compliance.sol

```diff
@@ -14,18 +14,18 @@ contract Compliance is ICompliance, Ownable { // TODO consider renaming to somet
      * External methods
      */
 
-    function getValidatorCompliance(address addr) external view returns (string memory conformanceType) {
+    function getValidatorCompliance(address addr) external view returns (string memory complianceType) {
```

Comment:
for short strings (up to 32 bytes) i think it's cheaper to represent as bytes32. and it gives a natural cap on the storage used - guards against missus. I know we did not consider integers yet, just making the comment... this approach can also be useful for `ContractRegistry` identifiers/names. 

they also compare easily etc.. 

https://ethereum.stackexchange.com/questions/11556/use-string-type-or-bytes32

@OdedWx can we ask Leonid if this approach (using bytes32 instead of string) is recommended?

Reply:
No longer relevant as we plan to represent as boolean